templates/ZF/mnc_5stage/train.prototxt

```diff
@@ -763,7 +763,7 @@ layer {
 		lr_mult: 2.0 
 	}
 	inner_product_param {
-		num_output: 8
+		num_output: {{ (num_classes|int) * 4 }}
```

Comment:
That was necessary because `num_classes` was being treated as a string and `num_output` was repeating that value 4 times. So, if `num_classes = 2`, `num_output = 2222` instead of 8. I have fixed that in the Python script, like you proposed.